src/system/UsersModule/Twig/Extension/ProfileExtension.php

```diff
@@ -152,6 +152,10 @@ private function determineProfileLink($userId = null, $userName = null, $class =
             return $userDisplayName;
         }
 
-        return '<a' . $class . ' title="' . ($this->translator->__('Profile')) . ': ' . htmlspecialchars($userDisplayName, ENT_QUOTES) . '" href="' . $href . '">' . $show . '</a>';
+        if (empty($title)) {
+            $title = $this->translator->__('Profile');
+        }
+
+        return '<a' . $class . ' title="' . ($title) . ': ' . htmlspecialchars($userDisplayName, ENT_QUOTES) . '" href="' . $href . '">' . $show . '</a>';
```

Comment:
htmlspecialchars is needed for replacing the quotes in `$title`, similar to `$userDisplayName`.